README.md

```diff
@@ -72,6 +72,36 @@ Options:
   -V, --version                        Print version
 ```
 
+Running using command line prints also the parameters of the network before and IPS. Parameters are printed in the following format:
+
+```
+Statistics for final network:
+----------------------------------------
+
```

Comment:
[opinion sharing]:
this may be a subjective thing, but I would find it more pleasing to my eyes without the newline here from the UI perspective

Reply:
👍